src/cmake/dependencies.cmake

```diff
@@ -90,6 +90,11 @@ if(BUILD_ASSIMP_SUPPORT)
   find_package(Assimp REQUIRED)
 endif()
 
+# v-hacd
+set(NO_OPENCL ON CACHE BOOL "NO_OPENCL" FORCE)
+set(NO_OPENMP ON CACHE BOOL "NO_OPENMP" FORCE)
+add_subdirectory("${DEPS_DIR}/v-hacd/src/VHACD_Lib")
```

Comment:
It might be useful to add a comment explaining why `src/VHACD_Lib`  was added as a subdirectory and not `src/`.

(I checked the sources and it makes sense to do so, in order to skip the tests, but explicitly pointing this out could help debugging in case V-HACD's top-level CMakeLists becomes important at some point in the future.)